packages/next/README.md

```diff
@@ -1,3 +1,86 @@
+<p align="center">
+  <a href="https://trpc.io/"><img src="../../www/static/img/logo-text.svg" alt="tRPC" height="130"/></a>
+</p>
+
+<p align="center">
+  <strong>End-to-end typesafe APIs made easy</strong>
+</p>
+
+<p align="center">
+  <!-- TODO: replace with new version GIF -->
+  <img src="https://storage.googleapis.com/trpc/trpcgif.gif" alt="Demo" />
+</p>
+
 # `@trpc/next`
 
-https://trpc.io
+> The `@trpc/next` package is responsible for connecting a trpc server to a Next.js application.
+
+## Documentation
+
+Full documentation for `@trpc/next` can be found [here](https://trpc.io/docs/nextjs)
+
+## Installation
+
+```bash
+# NPM
+npm install @trpc/next @trpc/react react-query
+
+# Yarn
+yarn add @trpc/next @trpc/react react-query
+
+# Pnpm
+pnpm install @trpc/next @trpc/react react-query
+```
+
+## Basic Example
```

Comment:
I'm unsure about having these examples here to be honest. It's another thing that has to be kept up to date alongside the docs.

Reply:
I think it's very minimal and small that it won't be hard to keep updated and gives the user some kind of gist about the package.